plugins/module/create_kafka_acl_binding.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+import random
+import time
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_kafka_acl_binding
+
+short_description: Create Access Control Lists (ACLs) for Red Hat OpenShift Streams for Apache Kafka Instance
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create Access Control Lists (ACLs) Red Hat OpenShift Streams for Apache Kafka Instance. More details can be found [here](https://github.com/redhat-developer/app-services-sdk-python/blob/main/sdks/kafka_instance_sdk/docs/AclBinding.md)
+
+options:
+    principal:
+        description: ID of the Service Account to bind created ACLs to
+        required: true
+        type: str
+    kafka_id:
+        description: ID of the Kafka instance
+        required: true
+        type: str
+    kafka_admin_url:
```

Comment:
Why we need it when we have kafka id? If we have id we can fetch url?

Reply:
Yea. That's what I'm doing. I'm making the module as decoupled as I can. If one already has a kafka or wants to add new permission to service account. Then this is needed?

Reply:
IMHO We miss requirements by implementing it this way. 
In Epic we say that user should be able to create and configure kafka so it can pass credentials in secure way to other steps of ansible and terrform... yet both providers making it increasingly difficult to use it this way.

What is the point of hardcoding kafka url where the point for using Ansible (and terraform) is to actually get that url dynamically :)

Reply:
We can still get the URL dynamically. That isn't an issue, the URL is not required. We can retrospectively create new ACL bindings. This just gives an extra option. you can leave this blank and nothing happens (so longs as you pass the ID). But if you want to you can pass the URL and we get the bonus of one less API call

Reply:
IMHO this is confusing as there are 2 values that are mutually exclusive and no validation when both approved. 
Let's chat about this next week. I must be missing something obvious here. 

I see it this way:

1. Create Kafka
2. Create Service accounts 


                   3. Get Kafka ID or URL 
                   4. Get Service Account Details
                   5. Create ACL using Kafka ID/Service

6. Continue with other steups using URL, Service account details.

We need to be able to:
- Wait for kafka 
- Pass all values dynamically in secure way. 

I do not see this (probably need whole ansible role done to see how this could work with current setup.